client/src/client.rs

```diff
@@ -476,7 +487,33 @@ impl<DB: Database> Client<DB> {
         };
     }
 
+    pub async fn get_last_checkpoint(&self) -> Option<String> {
```

Comment:
this doesn't seem related, do you want to include it?

Reply:
I use it in the ethereum-canister. Do you prefer it in a separate PR?

Reply:
not a big deal for me but I'd prefer separate PR, this feels a bit sneaky